backlogger.py

```diff
@@ -230,7 +234,10 @@ def render_influxdb(data):
             else:
                 initialize_md(data)
                 with open("index.md", "a") as md:
-                    for row in render_table(data):
+                    all_good, rows = render_table(data)
+                    for row in rows:
                         md.write("|".join(row) + "\n")
     except FileNotFoundError:
         sys.exit("Configuration file {} not found".format(switches.config))
+    if switches.exit_code and not all_good:
```

Comment:
Wouldn't this always return with exit code 3 in the `switches.output == "influxdb"` case (if `switches.exit_code` is set)?

Reply:
Right. I just doubled-checked it's only used in the markdown case - so this would only work if @baierjan plans to use this in our Salt-based GitLab pipelines 🤔

Reply:
> Wouldn't this always return with exit code 3 in the `switches.output == "influxdb"` case (if `switches.exit_code` is set)?

Hm, right. I didn't notice the second branch which will not set proper default value for the `all_good`. On the other hand, runs with influxdb output will likely not use `--exit-code` switch, so it will end up with exit code 0.

I will fix it, so one can use `--exit-code` and `--output influxdb` together.